frontend/src/pages/instance/components/InstanceForm.vue

```diff
@@ -28,14 +43,13 @@
                 data-form="project-name"
             >
                 <template #default>
-                    Application Name
+                    Instance Domain
```

Comment:
```suggestion
                    Instance Name
```

Lets keep calling it the name - domain doesn't feel right in this context. We can update the description text to reassert the connection between name and the sub-domain the instance is then accessed on... but I have to do that as a separate suggestion is it spans two different changes in the the diff...

Reply:
@joepavitt Asked for this to be changed to domain for clarity, @joepavitt do you have further insights here?

Reply:
> We currently happen to then use that name as part of the url to access it, but the identify of the thing is its name.

Currently, it's a domain, I want that clearly shown to the user that's it's the domain too.

Alternatively, if we do change this to "Name", then I vote we clearly include a "sample url" or something like that in the label/helper text of that field that shows it'll be their domain too.